Valet/VALValet.m

```diff
@@ -450,6 +450,10 @@ - (BOOL)setObject:(nonnull NSData *)value forKey:(nonnull NSString *)key options
             keychainData[(__bridge id)kSecValueData] = value;
             
             status = VALAtomicSecItemAdd((__bridge CFDictionaryRef)keychainData, NULL);
+            
+            if (status == errSecMissingEntitlement) {
```

Comment:
Fair re the check condition. Happy to leave it as an NSLog.

After taking a closer look at the apple link forum, this does seem different from #4 in the README. Mind writing up a #5? Doesn't need to be a blocker, but a fast-follow would be appreciated.

As for `objectForKey:options:`, do let me know!